docs/getting_started.rst

```diff
@@ -0,0 +1,131 @@
+:orphan:
+
+###############
+Getting started
+###############
+
+Installation
+============
+
+Qiskit Nature depends on the main Qiskit package which has its own
+`Qiskit Getting Started <https://qiskit.org/documentation/getting_started.html>`__ detailing the
+installation options for Qiskit and its supported environments/platforms. You should refer to
+that first. Then the information here can be followed which focuses on the additional installation
+specific to Qiskit Nature .
```

Comment:
```suggestion
specific to Qiskit Nature.
```

Reply:
The current spell check uses pylint which only looks at python source code. There is a PR on hold on Optimization to posssibly use sphinxcontrib.spelling which is called from sphinx and I believe would look at the whole documentation.